src/payment/checkout/payment-form/CardHolderInformation.jsx

```diff
@@ -399,6 +399,6 @@ CardHolderInformationComponent.defaultProps = {
 };
 
 export default connect(
-  () => {},
+  null,
```

Comment:
So this was okay but the other wasn't!?  Wacky. O_o

Reply:
Yeah, dunno.